eng/common/build.ps1

```diff
@@ -33,7 +33,7 @@ Param(
 # Unset 'Platform' environment variable to avoid unwanted collision in InstallDotNetCore.targets file
 # some computer has this env var defined (e.g. Some HP)
 if($env:Platform) {
-  $env:Platform=""  
+  $env:Platform=""
```

Comment:
I don't think we should touch the file at all - otherwise Arcade updates could cause merge conflicts.
In addition - these are just whitespace changes, so not needed.

Reply:
Yeah, looks like VS Code got a bit too aggressive with the whitespace changes.